waspc/data/Generator/templates/react-app/tsconfig.node.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "composite": true,
+    "skipLibCheck": true,
```

Comment:
@Martinsos Why did we. do this change?

Reply:
New vite adds that when you scaffold new project with it. I read about what it is, and was convinced that it makes sense during development, as it can speed things up.
I am ok with removing it, but if that is their default, I concluded it makes sense we also have it as default.

Reply:
If it's the default for Vite, let's keep it.